sktime/forecasting/model_selection/_tune.py

```diff
@@ -311,8 +356,10 @@ class ForecastingGridSearchCV(BaseGridSearch):
         None means 1 unless in a joblib.parallel_backend context.
         -1 means using all processors.
     refit: bool, optional (default=True)
-        True = refit the forecaster with the best parameters on the entire data in fit
-        False = best forecaster remains fitted on the last fold in cv
+        True = refit the forecaster with the best parameters on the entire data in fit.
+        False = best forecaster remains fitted on the last fold in cv but will b
+        updated to move cutoff to present by means of update(update_params=False).
```

Comment:
I think you should make it a little clearer what you mean regarding  "move cutoff to present".  I think it was clear what you intended from discussion on the PR, but think its a little more confusing here without that context.